source/MaterialXFormat/File.h

```diff
@@ -207,6 +207,11 @@ class MX_FORMAT_API FilePath
     /// Return the directory containing the executable module.
     static FilePath getModulePath();
 
+    #if defined(__APPLE__)
+    /// Return the directory containing the shared library.
+    static FilePath getSharedLibraryPath();
```

Comment:
This ends up being an unusual addition to the `FilePath` class, where a new public method is only defined on a _single platform_, and I wonder if we can improve upon this.

Could the logic within `getSharedLibraryPath` instead be moved directly into `getDefaultDataSearchPath`, as a block of code that is included within a check of `BUILD_APPLE_FRAMEWORK`?

Further in the future, if this logic needs to be shared between `getDefaultDataSearchPath` and other methods, we can design a good approach for making it available more broadly, but I feel that the current approach is not quite what we want here.

Reply:
That's fair. I inlined the method as requested.